UPDATING.md

```diff
@@ -87,6 +87,17 @@ configuration, so creating EMR clusters might fail until your connection is upda
 Ec2SubnetId, TerminationProtection and KeepJobFlowAliveWhenNoSteps were all top-level keys when they
 should be inside the "Instances" dict)
 
+### LDAP Auth Backend now requires TLS
+
+Connecting to an LDAP serever over plain text is not supported anymore. The
+certificate presented by the LDAP server must be signed by a trusted
+certificiate, or you must provide the `cacert` option under `[ldap]` in the
+config file.
+
+If you want to use LDAP auth backend without TLS then you will habe to create a
+custom-auth backend based on
+https://github.com/apache/incubator-airflow/blob/1.10.0/airflow/contrib/auth/backends/ldap_auth.py
```

Comment:
How to configure this backend ?

Reply:
It is not configurable - if you want the old behaviour back you need to take this code, make it available in your install and configure the auth backend for the webserver to use it.

Reply:
So my question one more time.
Why we created not backward compatible change in bugfix upgrade ?

Reply:
Why we did a breaking change without option to configure this ? 
Not everyone is using TLS enabled LDAP.
Upgrade from 1.10.0 to 1.10.1 completely broke my Airflow instalation. 
And this should be bugfix release.